src/components/homepage.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+import { IoSearch } from 'react-icons/io5';
+import { countriesFetch, continents } from '../redux/countrySlice';
+import worldmap from '../assets/worldmap.svg';
+import Navbar from './Nav';
+import CountrySection from './countrysection';
+
+const Home = () => {
```

Comment:
- [ ] Hi @SabaAhmad404, let's kindly note that this is a _mobile web app_ and we are requested to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

* Typography: font face, size, and weight
* Layout: composition and space between elements.

**Screenshots:**

| Original | Yours |
| --- | --- |
| ![CaptureO](https://user-images.githubusercontent.com/63087491/219634183-bc82bb8a-fde4-4e34-9e30-7d91f5753b50.PNG) | ![Capture](https://user-images.githubusercontent.com/63087491/219634363-c9ab7869-a5c2-4fc9-97dc-6ae27f1e28e2.PNG) |

**NB:** Having a nice design increases your chances for a job as recruiters will be checking your GitHub projects.